src/SelfScreener/AsymptomaticFlowIntro.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { FunctionComponent } from "react"
```

Comment:
Thoughts on simplifying this flow and navigating directly to the guidance screen?